conformance/ConformanceApi.fsd

```diff
@@ -212,8 +212,8 @@ service ConformanceApi
 	{
 		[http(from: query), required] query: string;
 		[required] normal: string;
+		[validate(count: 5)] fiveWidgets: Widget[];
 		widget: Widget;
-		widgets: Widget[];
```

Comment:
We should avoid breaking changes to the conformance API. See [ConformanceTests.json](https://github.com/FacilityApi/FacilityCSharp/blob/master/conformance/ConformanceTests.json) for how it is used.

How about a new optional field: `[validate(count: 2)] point: double[];`